ddspipe_participants/include/ddspipe_participants/writer/dds/CommonWriter.hpp

```diff
@@ -128,20 +131,30 @@ class CommonWriter : public BaseWriter
     // INTERNAL METHODS
     /////////////////////////
 
-    virtual
-    fastdds::dds::PublisherQos
-    reckon_publisher_qos_() const noexcept;
+    DDSPIPE_PARTICIPANTS_DllAPI
+    virtual fastdds::dds::PublisherQos reckon_publisher_qos_() const noexcept;
+
+    DDSPIPE_PARTICIPANTS_DllAPI
+    virtual fastdds::dds::DataWriterQos reckon_writer_qos_() const noexcept;
```

Comment:
missing documentation

Reply:
I know, many classes and methods have no documentation. In my opinion, this should be done in a dedicated PR.